draft-ietf-cbor-update-8610-grammar.md

```diff
@@ -523,6 +452,98 @@ applied in the present document.
 {: #collected-abnf title="ABNF for CDDL as updated"
 sourcecode-name="cddl-updated-complete.abnf"}
 
+# Details about Covered Errata Reports {#Err6543-covered}
+
+This appendix is informative.
+
+{{Err6543}} observes that
+the ABNF used in {{RFC8610}} for the content of byte string literals
+lumps together byte strings notated as text with byte strings notated
+in base16 (hex) or base64 (but see also updated BCHAR rule in {{e6527-new2}}):
+
+~~~ abnf
+; RFC 8610 ABNF:
+bytes = [bsqual] %x27 *BCHAR %x27
+BCHAR = %x20-26 / %x28-5B / %x5D-10FFFD / SESC / CRLF
+~~~
+{: #e6527-orig2a title="Original RFC 8610 ABNF for BCHAR"}
+
+## Change proposed by Errata Report 6543
+{:unnumbered}
+
+Errata report 6543 proposes to handle the two cases in separate
+ABNF rules (where, with an updated SESC, BCHAR obviously needs to be
+updated as above):
+
+~~~ abnf
+; Err6543 proposal:
+bytes = %x27 *BCHAR %x27
+      / bsqual %x27 *QCHAR %x27
+BCHAR = %x20-26 / %x28-5B / %x5D-10FFFD / SESC / CRLF
+QCHAR = DIGIT / ALPHA / "+" / "/" / "-" / "_" / "=" / WS
+~~~~
+{: #e6543-1 title="Errata Report 8653 Proposal to Split the Byte String Rules"}
+
+This potentially causes a subtle change, which is hidden in the WS rule:
+
+~~~ abnf
+; RFC 8610 ABNF:
+WS = SP / NL
+SP = %x20
+NL = COMMENT / CRLF
+COMMENT = ";" *PCHAR CRLF
+PCHAR = %x20-7E / %x80-10FFFD
+CRLF = %x0A / %x0D.0A
+~~~
+{: #e6543-2 title="ABNF definition of WS from RFC 8610"}
+
+This allows any non-C0 character in a comment, so this fragment
+becomes possible:
+
+~~~ cddl
+foo = h'
+   43424F52 ; 'CBOR'
+   0A       ; LF, but don't use CR!
+'
+~~~
+
+The current text is not unambiguously saying whether the three apostrophes
+need to be escaped with a `\` or not, as in:
+
+~~~ cddl
+foo = h'
+   43424F52 ; \'CBOR\'
+   0A       ; LF, but don\'t use CR!
+'
+~~~
+
+... which would be supported by the existing ABNF in {{-cddl}}.
+
+## No change needed after addressing Err6527 and Err6278 ({{e6527}})
```

Comment:
The structure was confusing to me at first: We start (above the first subsection heading in the appendix) with 6543, which feels off but OK, then comes the change proposed by 6543, but here I'm left hanging with "OK, but what's the conclusion here? Did we follow the proposed change or not? Sounds like not." … but the next subsection apparently goes off with different errata items.

With reading it again and jumping back to see that the errata interact, I now do see that this is still about 6543, but on how those other errata resolutions solved it.

I don't have any local suggestions on how to avoid more readers doing the same exercise; maybe a combination of:
* The whole Appendix B is all about 6543. That could be reflected in the appendix title. (IIUC all errata handling unrelated to 6543 has little enough need for discussion to be done inline).
* The sub-headings may then be "Change proposed in the errata report" and "No change needed due to updated string literal grammar (Section 2.1)" ("…Figure 2 and Figure 4, *as prompted by Err6527 and Err6278*.")

Reply:
I prefer to be explicit as in "Change Proposed By Errata Report 6543", but otherwise put in your suggestions.